scripts/install-az-cli.sh

```diff
@@ -0,0 +1,17 @@
+sudo apt-get update
+sudo apt-get install apt-transport-https ca-certificates curl gnupg lsb-release
+
+sudo mkdir -p /etc/apt/keyrings
+curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
+  gpg --dearmor | sudo tee /etc/apt/keyrings/microsoft.gpg > /dev/null
+sudo chmod go+r /etc/apt/keyrings/microsoft.gpg
+
+AZ_DIST=$(lsb_release -cs)
+echo "Types: deb
+URIs: https://packages.microsoft.com/repos/azure-cli/
+Suites: ${AZ_DIST}
+Components: main
+Architectures: $(dpkg --print-architecture)
+Signed-by: /etc/apt/keyrings/microsoft.gpg" | sudo tee /etc/apt/sources.list.d/azure-cli.sources
+
+sudo apt-get install azure-cli=${AZ_CLI_VERSION:-2.61.0}-1~${AZ_DIST}
```

Comment:
Given the default AZ_CLI_VERSION here could you not set it in the workflows?

Reply:
I could, but I think it's useful documentation for someone looking at the workflows to see we're installing a specific version.